### ISSUE TITLE
[Usable – Make Code – Home page]  There is no chevron or underline for footer link controls under ‘Home page’.

### ISSUE THREAD

**Issue by Shalini3457** [microsoft/pxt-microbit#3434]
**User Experience:**
User will be misguided If any actionable control(link) doesn't have underline or chevron as they will skip the functionality of such link. Not only this, user will face disappointment if visually appearing link display as text as user won't be proceed further for relevant action.

**Note**: User credentials should NOT be included in the bug.

**Repro steps:**
1. Launch application URL : https://makecode.microbit.org/# 
2. Home page will open.
3. Navigate to the controls appearing in Home page section.  
4. Verify visual 'Microsoft Make Code, Terms of use, Privacy' link by using Developer tool(F12).

**Actual Result:**
There is no chevron or underline appears for visual 'Microsoft Make Code, Terms of use, Privacy' link controls under ‘Home page’ and due to this, user is getting affected as he/she might consider that link as text and will skip the functionality of such link.

**Expected Result:**
There must be chevron or underline on visual 'Microsoft Make Code, Terms of use, Privacy' link controls under ‘Home page’, so that user could easily navigate that link to get more details.

**Note:**
This issue also exists with Firefox and Chrome Browser.

**Reference Links:**
•	External Bug Process: If this bug belongs to external team, mark it as resolved/done and assign it back to the tester with notes on where to file/route the bug. For more information please use this [Link](https://microsoft.sharepoint.com/teams/ceacc/SitePages/External-Bug-process.aspx). “HCL Staff should not log any third party external bugs. Should be routed to EDAD team”.
•	Please reach out to [C&AI Teams channel](https://teams.microsoft.com/dl/launcher/launcher.html?url=%2F_%23%2Fl%2Fteam%2F19%3A3cd071821b694f11b054d878453cd75f%40thread.skype%2Fconversations%3FgroupId%3D30e3f968-15dd-4da2-bd8e-85f27e4af2c3%26tenantId%3D72f988bf-86f1-41af-91ab-2d7cd011db47&type=team&deeplinkId=99c73412-3928-459b-a051-ad90788e6969&directDl=true&msLaunch=true&enableMobilePage=true&suppressPrompt=true) for any process related queries.

**Test Environments:**
Version: 20H2 (OS Build: 19042.508)
Browser: Edge chromium DEV Build (Version 87.0.654.0), Firefox(75.0), Chrome
#URL:https://makecode.microbit.org/#

**Comment by Shalini3457** [comment]
![Usable_Link does not have underline](https://user-images.githubusercontent.com/72871089/96090444-d348df00-0ee5-11eb-82ff-4ba707bba3ec.jpg)
